app/views/pages/use-cases/using-rest-endpoints-with-react.liquid

```diff
@@ -0,0 +1,302 @@
+---
+converter: markdown
+metadata:
+  title: Using REST endpoints with a React app
+  description: This use case describes how to use platformOS as a REST backend service for a React app.
+---
+
+## Problem
+
+How to use platformOS as a REST backend for your React app? The goal is to easily define new endpoints for your React app and have your app ready for local development communication with your instance via proxy.
+
+This article is based on React version 17.
+
+To follow this article, you should have your development environment and a platformOS Instance set up, be familiar with pages, layouts and React as well.
+
+- [Get Started](/get-started)
+- [Pages and Layouts](/developer-guide/pages/pages)
+- [Creating a React app](https://reactjs.org/docs/create-a-new-react-app.html)
+
+## Solution
+
+Using platformOS as a REST api for your React app is a eight-step process.
+
+<div data-autosteps></div>
+
+### Step 1: Create place for React app files
+
+The source files of a web app are not part of platformOS and can be stored anywhere you want.
+In this example, to keep it together in one repo and in one project, you'll create the React app directly in your instance's folder but not inside the `/app` folder because it is not recommended to deploy your source files to the server, only the built and minified ones.
+
+Navigate to your instance folder and create new app with:
+
+`npx create-react-app my-app --template typescript`
```

Comment:
I would suggest using a different name than `my-app`

1. We already have a top-level `app` folder. 
2. Later on in the article you suggest adding `app/assets/web-app` folder, which is yet another name.

I don’t have a good replacement, but I believe that `react-app` would at least be more descriptive of that’s inside.